### PR TITLE
Remove variables for testnet v1.

### LINKED ISSUE #622: Remove variables for old testnet

### PR DESCRIPTION
**Pull request by jcvernaleo**
Closes #622

### CODE REVIEW

File: cmd/addblock/config.go [hunk 1]
```diff
@@ -112,14 +112,14 @@ func loadConfig() (*config, []string, error) {
...
-		activeNetParams = &chaincfg.TestNetParams
+		activeNetParams = &chaincfg.TestNet2Params
 	}
 	if cfg.SimNet {
 		numNets++
 		activeNetParams = &chaincfg.SimNetParams
 	}
 	if numNets > 1 {
-		str := "%s: The testnet, regtest, and simnet params can't be " +
+		str := "%s: The testnet2, regtest, and simnet params can't be " +
```
Comment (jrick) [inline cmd/addblock/config.go]: "testnet" is the actual name of the parameter

File: cmd/findcheckpoint/config.go [hunk 2]
```diff
@@ -99,14 +99,14 @@ func loadConfig() (*config, []string, error) {
...
-		activeNetParams = &chaincfg.TestNetParams
+		activeNetParams = &chaincfg.TestNet2Params
 	}
 	if cfg.SimNet {
 		numNets++
 		activeNetParams = &chaincfg.SimNetParams
 	}
 	if numNets > 1 {
-		str := "%s: The testnet, regtest, and simnet params can't be " +
+		str := "%s: The testnet2, regtest, and simnet params can't be " +
```
Comment (jrick) [inline cmd/findcheckpoint/config.go]: "testnet" is the actual name of the parameter

File: database/cmd/dbtool/globalconfig.go [hunk 3]
```diff
@@ -68,14 +68,14 @@ func setupGlobalConfig() error {
...
-		activeNetParams = &chaincfg.TestNetParams
+		activeNetParams = &chaincfg.TestNet2Params
 	}
 	if cfg.SimNet {
 		numNets++
 		activeNetParams = &chaincfg.SimNetParams
 	}
 	if numNets > 1 {
-		return errors.New("The testnet and simnet params can't be " +
+		return errors.New("The testnet2 and simnet params can't be " +
```
Comment (jrick) [inline database/cmd/dbtool/globalconfig.go]: "testnet" is the actual name of the parameter

File: cmd/addblock/config.go (hunk 1 above)
Comment (jcvernaleo) [inline cmd/addblock/config.go]: fixed

File: cmd/findcheckpoint/config.go (hunk 2 above)
Comment (jcvernaleo) [inline cmd/findcheckpoint/config.go]: fixed

File: database/cmd/dbtool/globalconfig.go (hunk 3 above)
Comment (jcvernaleo) [inline database/cmd/dbtool/globalconfig.go]: fixed

**Pull request merged**